grammar.js

```diff
@@ -152,6 +159,7 @@ module.exports = grammar({
     top_level_object: $ => seq($._declaration, optional($._semis)),
 
     type_alias: $ => seq(
+      optional($.modifiers),
```

Comment:
@colleend that's the thing I mentioned in your PR to fwcd that you could also include (small change).
Better to not have too many PRs, so for small changes  like that I think it's ok we pack multiple small changes in only 1 PR.

Reply:
I see, good find! I can put this fix in my PR then!